src/VisualStudio/Core/Def/Implementation/MoveStaticMembers/MoveStaticMembersDialog.xaml

```diff
@@ -53,30 +52,65 @@
                 Text="{Binding DestinationName, UpdateSourceTrigger=PropertyChanged}"
                 Margin="0 0 20 0"/>
```

Comment:
With the other controls in group boxes, this textbox feels too far to the over. It might be nice to add some left margin in order to have it more closely align with the other controls.